### PR TITLE
working on price stats

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

File: js/stats.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      )
+        .then((res) => res.json())
+        .then((data) => {
+          const price = parseFloat(data.priceUSD).toFixed(14);
+          document.getElementById("coin-price").innerText = price;
+        });
+        return coin_price;
+
+  };
+
```
Comment (bobachef) [inline js/stats.js:14]: This is how it should look
```
async function price () {
  const response = await fetch('https://api.dex.guru/v1/tokens/0x3504de9e61fdff2fc70f5cc8a6d1ee493434c1aa-bsc')
  const data = response.json()
  const price = parseFloat(data.priceUSD).toFixed(14)
  document.getElementById('coin-price').innerText = price
  return price
}
```

**Review by bobachef — CHANGES_REQUESTED** [review]
async await pattern means no .then .catch

File: js/stats.js (hunk 1 above)
Comment (Shaharyar-saleem) [reply js/stats.js:14]: ok 
got it

**Comment by bobachef** [comment]
You're code doesn't work because you're not calling the function.

image

File: js/stats.js (hunk 1 above)
Comment (bobachef) [reply js/stats.js:14]: you also need to call the function.. cuz it's broken.

Comment (Shaharyar-saleem) [reply js/stats.js:14]: i am trying this format and this is not returning the desire data

Comment (bobachef) [reply js/stats.js:14]: are you calling the function?

File: index.html [hunk 2]
```diff
@@ -107,7 +107,6 @@
                                 <div class="coin-value-card" data-Speed="3000" data-aos="flip-right"
                                     data-aos-once="ture">
                                     <h5 class="" id="coin-price"></h5>
-                                    <!-- 0.00000000753379 -->
```
Comment (Shaharyar-saleem) [inline index.html:110]: removed

**Review by bobachef — APPROVED** [review]
async await pattern still incorrect but not gonna waste more time on it

**Pull request merged**